packages/subscriptions/graphql/resolvers/_queries/notifications.js

```diff
@@ -11,6 +11,7 @@ module.exports = async (_, args, context) => {
     {
       userId: me.id,
       ...(args.onlyUnread ? { readAt: null } : {}),
+      ...(args.filter ? { eventObjectType: args.filter } : {}),
     },
```

Comment:
```suggestion
    {
      ...(args.onlyUnread ? { readAt: null } : {}),
      ...(args.filter ? { eventObjectType: args.filter } : {}),
      userId: me.id,
    },
```

This code makes me a wee bit nervous. I had to go and read the definition of the `EventObjectType` to makes sure it does not conatina and overwirte the `userId`.

Reply:
And I see how what I wrote makes no senese ;)

Still getting used to using the spread operator to add a conditional property.